src/V1/Actor/Template/AnnotationTokenizer.php

```diff
@@ -17,6 +17,8 @@ class AnnotationTokenizer implements AnnotationTokenizerInterface
 
     protected $TokenizedContents;
 
+    private $memoizedAnnotationProcessors = [];
```

Comment:
I'm willing to strongly type this by injecting an `AnnotationProcessor\Map\Factory`, but because it's a strictly internal property, I wasn't too concerned about it.